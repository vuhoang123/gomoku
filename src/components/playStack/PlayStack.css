.play-stack{
  padding: 15px;
}
.play-stack button{
  margin-right: 10px;
}
