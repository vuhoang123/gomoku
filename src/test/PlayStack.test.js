
import PlayStack from "../components/playStack";
import renderer from 'react-test-renderer';

test('Check play stack', () => {
  const tree = renderer.create(
    <PlayStack content={[]} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
