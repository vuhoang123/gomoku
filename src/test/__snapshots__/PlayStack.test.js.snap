// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check play stack 1`] = `
<div
  className="play-stack"
>
  <button
    onClick={[Function]}
  >
    Sort Asc
  </button>
  <button
    onClick={[Function]}
  >
    Sort Desc
  </button>
  <button
    onClick={[Function]}
  >
    Sort By Time
  </button>
  <p>
    Step sorted by 
    asc
  </p>
</div>
`;
